test/browser/component-tests.js

```diff
@@ -2,170 +2,170 @@
 /* global sinon, expect */
 /* eslint no-unused-expressions:0 */
 
-describe(`Simple Component instance`, function() {
-  var el;
+describe(`Simple Component instance`, () => {
```

Comment:
there's a reason not to use arrow functions in mocha tests. `this` is often uses to share state across tests and can break some things.

Reply:
https://mochajs.org/#arrow-functions